workflow/scripts/copy_graph_plots.py

```diff
@@ -0,0 +1,23 @@
+from contextlib import redirect_stderr
+
+with open(snakemake.log[0], "w") as logfile:
+    with redirect_stderr(logfile):
+        logfile.write("A")
```

Comment:
Care for something more descriptive than `A`? :smile:

Reply:
Ah, the classic debug logging :D
We can either remove these or replace with "initialize copying quality control plots" etc

Reply:
Removed for now. If we need any debug logging in the future, we can always bring such statements back, and even include more relevant info... ;)